addons/api/mirage/route-handlers/auth.js

```diff
@@ -33,12 +33,13 @@ export function authHandler({ scopes, authMethods }, request) {
     const scope = scopes.find(authMethod.scopeId);
     const scopeAttrs =
       this.serialize(scopes.find(scope.id));
+    const user = users.first();
```

Comment:
Would it make more sense to pass in a specific user rather than a list from which we assume there is a first user?